Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceContext.swift

```diff
@@ -0,0 +1,307 @@
+// swiftlint:disable force_try
+import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+class SyncPerformanceContext {
+    var client: StitchAppClient?
+    var mongoClient: RemoteMongoClient?
+    var coll: RemoteMongoCollection<Document>?
+
+    var dbName = ""
+    var collName = ""
+    var userId = ""
+
+    let joiner = ThrowingCallbackJoiner()
+    let streamJoiner = StreamJoiner()
+    let testParams: TestParams?
+    let transport: FoundationInstrumentedHTTPTransport?
+    let harness: SyncPerformanceIntTestHarness?
+
+    init(harness: SyncPerformanceIntTestHarness,
+         testParams: TestParams,
+         transport: FoundationInstrumentedHTTPTransport) throws {
+        self.testParams = testParams
+        self.transport = transport
+        self.harness = harness
+        if testParams.stitchHostName == "https://stitch.mongodb.com" {
+            print("PerfLog: Context: Initializing STITCH-Prod")
+            dbName = harness.stitchTestDbName
+            collName = harness.stitchTestCollName
+            client = harness.outputClient
+            mongoClient = harness.outputMongoClient
+            coll = mongoClient?.db(dbName).collection(collName)
+            print("PerfLog: Context: Finished initializing STITCH-Prod")
+        } else {
+            print("PerfLog: Context: Initializing STITCH-Local")
+            dbName = ObjectId().oid
+            collName = ObjectId().oid
+
+            harness.setUp()
+            let app = try! harness.createApp()
+            _ = try! harness.addProvider(toApp: app.1, withConfig: ProviderConfigs.anon())
+            let svc = try! harness.addService(toApp: app.1, withType: "mongodb", withName: "mongodb1",
+                                         withConfig: ServiceConfigs.mongodb(name: "mongodb1", uri: harness.mongodbUri))
+            let rule = RuleCreator.mongoDb(database: dbName, collection: collName,
+                                           roles: [RuleCreator.Role(read: true, write: true)],
+                                           schema: RuleCreator.Schema())
+            _ = try! harness.addRule(toService: svc.1, withConfig: rule)
+
+            client = try! harness.appClient(forApp: app.0, withTransport: transport)
+            client?.auth.login(withCredential: AnonymousCredential(), joiner.capture())
+            let _: Any? = try joiner.value()
```

Comment:
[style] unnecessary type declaration

Reply:
If you dont have :Any? then .value() fails because it cannot infer the type